pkg/internal/transform/routes.go

```diff
@@ -31,14 +31,49 @@ const wildCard = "/**"
 // RoutesConfig allows grouping URLs sharing a given pattern.
 type RoutesConfig struct {
 	// Unmatch specifies what to do when a route pattern is not matched
-	Unmatch UnmatchType `yaml:"unmatch"`
+	Unmatch UnmatchType `yaml:"unmatched"`
```

Comment:
Thinking on a past discussion about the `unmatch` meaning, I think "unmatched" should be ok:
https://dictionary.cambridge.org/dictionary/english/unmatched